conda/install.py

```diff
@@ -225,21 +224,6 @@ def warn_failed_remove(function, path, exc_info):
         log.warn("Cannot remove, unknown reason: {0}".format(path))
 
 
-def exp_backoff_fn(fn, *args):
```

Comment:
For the purposes of this fix I propose we leave `exp_backoff_fn` in place. But what we should do is move the `import random` inside the `try` block at the top, after the attempts to import `conda.` modules (which need to be converted to relative imports). This will minimize the number of changes needed to restore standalone functionality and minimize risk of breaking other things.

(I'm making these comments just for our discussion; I'll be pushing my changes here when I have finalized them.)